src/lib.rs

```diff
@@ -25,18 +25,34 @@ use std::fmt::Debug;
 /// Navigator's default crystal clock for PWM, with a value of 24.5760 MHz
 const NAVIGATOR_PWM_XTAL_CLOCK_FREQ: f32 = 24_576_000.0;
 
+/// The `AxisData` struct represents a set of three floating-point values for the x, y, and z axes.
+///
+/// You can use with:
```

Comment:
Its hard to track this, I would recommend to remove the this `you can use` section, the documentation has a search bar has maintaining it will be more work.
https://patrickelectric.work/blog/2020/how-to-rock-first-tips/

Reply:
Ok. Removed.